config/enrichments/cross_navigation_config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "schema": "iglu:com.snowplowanalytics.snowplow.enrichments/cross_navigation_config/jsonschema/1-0-0",
+
+    "data": {
+        "enabled": false,
```

Comment:
On second thought, I am in favor of making it enrichment because it isn’t just some internal change in the Enrich. It adds new context to events therefore it is actually ‘enriching’ the event therefore it would be better to make it enrichment. You can ignore above comment.